.github/workflows/fips-checksums.yml

```diff
@@ -69,6 +69,13 @@ jobs:
       - name: save PR number
         run: echo ${{ github.event.number }} > ./artifact/pr_num
       - name: save artifact
+        if: github.server_url == 'https://github.com'
```

Comment:
The fips-checksums.yml is not used on GHE, so you can just simply update to v4 here. However it might be also necessary to update the code that downloads the artifact in fips-label.yml if that will no longer work properly.

Reply:
For fips-label.yml, do you mean these scripts?

https://github.com/openssl/openssl/blob/012353bdf21b98def920ac317b94c4a9ed501b79/.github/workflows/fips-label.yml#L26-L46

Reply:
BTW, how would one know this isn't used on GHE?  I don't see what indicates that...

Reply:
> BTW, how would one know this isn't used on GHE? I don't see what indicates that...

You would have to look into GHE to see which workflows are explicitly disabled there (or which workflows were actually run in the last CI run).

Reply:
... yeah, so that could change.  Considering I've already done the work, I don't see a point undoing it under those conditions.